recipe/meta.yaml

```diff
@@ -10,14 +10,15 @@ source:
   md5: 92dd7de2621a6ab6861cc9b689e86c3e
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record record.txt
   osx_is_app: True
```

Comment:
It does as we do not want to include the `setuptools`-based entry points. This will replace them with `conda`'s. In the long run, maybe using `pip` will be a better solution, but things are a bit broken on Windows ATM. See this [comment]( https://github.com/conda-forge/nbformat-feedstock/pull/3#issuecomment-239433797 ) for details.